client/cloudAD_service.go

```diff
@@ -0,0 +1,48 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/models"
+)
+
+func (sm *ServiceManager) CreateActiveDirectory(active_directory_id string, tenant string, description string, nameAlias string, cloudADAttr models.ActiveDirectoryAttributes) (*models.ActiveDirectory, error) {
```

Comment:
CreateCloudActiveDirectory

Update all other functions and name to add cloud as this could conflict in the future.